bootstrap/mod/fabric/build.gradle.kts

```diff
@@ -36,6 +36,16 @@ dependencies {
 
     modImplementation(libs.cloud.fabric)
     include(libs.cloud.fabric)
+
+    include("org.incendo:cloud-fabric:2.0.0-beta.11")
+    include("org.incendo:cloud-fabric:2.0.0-beta.10")
+    include("org.incendo:cloud-fabric:2.0.0-beta.9")
+    include("org.incendo:cloud-fabric:2.0.0-beta.8")
+    include("org.incendo:cloud-fabric:2.0.0-beta.7")
+    include("org.incendo:cloud-fabric:2.0.0-beta.6")
+    include("org.incendo:cloud-fabric:2.0.0-beta.5")
+    include("org.incendo:cloud-fabric:2.0.0-beta.4")
```

Comment:
curious, does this work correctly already? Or is it broken due to their mod descriptions not stating the correct supported version?